docs/clients/stow_user_interface.rst

```diff
@@ -0,0 +1,144 @@
++========+=====================+=================+===========+===================+=================================================================================================================================================================================================+
+| 0      | zero_angle          | get, set, save  | float     | rad               | The anglular position that the module considers as its 'zero' position.                                                                                                                         |
++--------+---------------------+-----------------+-----------+-------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| 1      | target_angle        | get, set, save  | float     | rad               | The angular postion of the stow postition with reference to the zero angle.                                                                                                                     |
++--------+---------------------+-----------------+-----------+-------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| 2      | target_acceleration | get, set, save  | float     | .. math:: rad/s^2 | The maximum acceleration allowed when moving to the stow position.                                                                                                                              |
++--------+---------------------+-----------------+-----------+-------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| 3      | sample_zero         | set             |           |                   | Sets the module's current postiion as the zero angle.                                                                                                                                           |
++--------+---------------------+-----------------+-----------+-------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| 4      | stow                | set             |           |                   |                                                                                                                                                                                                 |
```

Comment:
No notes for this one? Maybe just add something like "Setting this triggers the manual to stow"